recipes-extended/qemu-dm/qemu-dm.inc

```diff
@@ -41,6 +41,7 @@ SRC_URI += "file://compile-time-stubdom-flag.patch \
             file://block-remove-unused-block-format-support.patch \
             file://net-Remove-unused-network-options.patch \
             file://round-pci-region-sizes.patch \
+            file://0001-vnc-stubs-Allow-vnc-none.patch \
```

Comment:
Noting here, when we uprev OE  to kirkstone, we will be able to drop this patch as it's included in v6.2.0 (current qemu version in kirkstone)